tests/test_interfaces.py

```diff
@@ -778,6 +723,103 @@ def test_get_interface_mac_address(self):
 
         self.assertEqual(self.network_manager.get_interface_mac(interface_name), self.mac_address)
 
+    def test_get_interface_1_ap_1_mon_internet_interfaces(self):
```

Comment:
I'm not so sure I got your point here. since I want to make sure if I add the internet card to the `_active` set, the original algorithm for choosing ap/deauth is not break.

using the method `is_interface_valid` is to test whether the user has specified the right card for ap/deauth.

So if the whole process is correct for the method you mentioned, we'll get 3 cards, and I assert for getting three cards.